src/editor/main.js

```diff
@@ -169,18 +185,24 @@ button('delete', () => {
   radio.closest('.layer').remove();
 });
 button('export', async () => {
-  const url = await toUrl(controller.export(), true);
-
-  let a = document.createElement('a');
-  a.href = url;
-  a.download = 'maskable_icon.png';
-  document.body.appendChild(a);
-  a.click();
-  document.body.removeChild(a);
-
-  if (url.startsWith('blob:')) {
-    URL.revokeObjectURL(url);
-  }
+  const exportSizes = new FormData(document.forms['exportSizes']).getAll('sizes')
+    // @ts-ignore
+    .map(item => parseInt(item, 10));
```

Comment:
I think casting to a string will work here. Otherwise I'd prefer to use ts-expect-error.
```suggestion
    .map(item => parseInt(item as string, 10));
```

Reply:
![image](https://user-images.githubusercontent.com/1783160/103527093-82b0a480-4e8a-11eb-998f-0243c6fb973c.png)

I will update it with ts-expect-error